docs/content/preview/api/ysql/syntax_resources/grammar_diagrams.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Grammar Diagrams
+title: Grammar Diagrams YSQL
```

Comment:
Revert all changes - pretty sure this is an auto generated file, and we just have to live with it. Anyhow, don't fix in this PR

Reply:
yes was unsure about this file. Okay let me revert it.